router.js

```diff
@@ -0,0 +1,30 @@
+const router = require('express').Router();
+const path = require('path');
```

Comment:
Is this needed in your router file? I scanned for it a couple times but I didnt see it, usually you would only need this in your server